docs/checks.md

```diff
@@ -644,14 +644,15 @@ Additionally, points are reduced if certain write permissions are defined for a
 
 Risk: `High`  (known vulnerabilities)
 
-This check determines whether the project has open, unfixed vulnerabilities
-using the [OSV (Open Source Vulnerabilities)](https://osv.dev/) service. An open
-vulnerability is readily exploited by attackers and should be fixed as soon as
+This check determines whether the project has open, unfixed vulnerabilities 
+in its dependencies using the [OSV (Open Source Vulnerabilities)](https://osv.dev/) service.
```

Comment:
do we still want to report the vuln in the project itself? I'd say that we want, no?

Reply:
Added it back in.